lib/app.ts

```diff
@@ -27,6 +28,7 @@ const app = new Hono();
 app.use(compress());
 
 app.use(mLogger);
+app.use(trace);
```

Comment:
If [`DEBUG_INFO`](https://docs.rsshub.app/install/config#logging-configurations) is set to `false` or any strings other than `true`, the debug info from RSSHub will be hidden and only available if the same string is provided as query string `debug`.

`/metrics` does not respect this behaviour currently.

The same also apply to [`ACCESS_KEY`](https://docs.rsshub.app/install/config#access-keycode).
https://github.com/DIYgod/RSSHub/blob/15a51bdffb7515504e13bb10b36e70be504a8ca7/lib/middleware/access-control.ts#L15

Reply:
Basicly, metrics should be export with a external local port. In that case rsshub may handle another port with `127.0.0.1:9090` for example.
I'm not sure if an external port should be created, but it might be a better solution than modify if else for access-control.
And also should be with env to control local server enable or not.